php/tower/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "tower/tower",
+    "description": "The Tower Framework",
+    "keywords": ["framework", "tower" , "high performance"],
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Amir",
+            "email": "faramarzii.amir@gmail.com"
+        }
+    ],
+    "require": {
+        "php" : ">=8.0",
+        "tower/tower-framework": "^1.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\" : "app/" ,
+            "Database\\" : "database/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "post-root-package-install": [
+            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
+        ]
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require-dev": {
+        "fakerphp/faker": "^1.15",
+        "phpunit/phpunit": "^9.5"
+    }
```

Comment:
not sure this is useful. 

PS : in fact having as-slim-as it could be implementation will help me on deployment (those benchmarks will run on cloud in the future)